src/Enum.php

```diff
@@ -263,6 +280,9 @@ public function __toString(): string
      * Get a list of enumerator instances ordered by ordinal number
      *
      * @return static[]
+     *
+     * @psalm-return list<DummyEnum>
```

Comment:
I'm guessing this should refer to `<Enum>`

Reply:
Indeed! Lemme fix these :-)

Reply:
Force-pushed after getting rid of it here :+1: Was `@psalm-return list<static>`

Reply:
Right, makes sense.